Maintainer: Artjom Slepnjov "Shellgen"
Description: fix url response for the user.
Ver: fiche-0.9.1
Date: 2024-02-01 20:00 UTC
==============================================================
--- a/fiche.c
+++ b/fiche.c
@@ -652,11 +652,13 @@
 
     // Write a response to the user
     {
+        char *file_name = "index.txt";
+
         // Create an url (additional byte for slash and one for new line)
-        const size_t len = strlen(c->settings->domain) + strlen(slug) + 3;
+        const size_t len = strlen(c->settings->domain) + strlen(slug) + strlen(file_name) + 4;
 
         char url[len];
-        snprintf(url, len, "%s%s%s%s", c->settings->domain, "/", slug, "\n");
+        snprintf(url, len, "%s%s%s%s%s%s", c->settings->domain, "/", slug, "/", file_name, "\n");
 
         // Send the response
         write(c->socket, url, len);
